build/repo.targets

```diff
@@ -1,8 +1,12 @@
 <Project>
+  <!--
+    TODO include VSIX build in default lifecycle builds
```

Comment:
The plan is to have VS 2017 installed on all our build agents. We're avoiding the hassle of updating them later by installing a RC build now.